### DISCUSSION TITLE
Custom Tree Section, how to toggle checkbox

### DISCUSSION THREAD

**Discussion by npreiser** [redhat-developer/vscode-extension-tester#1515] [Q&A]
I have implemented a custom tree (on the side bar left) and using this tester I am able to select items within the tree, but am unable to check the box on the left of the items.  I have tried to "send keys" like spacebar to the item, but it gives me a item not interactable.   Any ideas how to check the check box?,

**Comment by djelinek** [comment]
could you please share the screenshot how the sidebar with checkboxes looks like?
